testsuite/features/support/cobbler_test.rb

```diff
@@ -46,6 +46,20 @@ def get_list(what)
     result
   end
 
+  def distro_create(name, kernel, initrd, breed = 'suse')
+    begin
+      distro_id = @server.call('new_distro', @token)
+      @server.call('modify_distro', distro_id, 'name', name, @token)
+      @server.call('modify_distro', distro_id, 'kernel', kernel, @token)
+      @server.call('modify_distro', distro_id, 'initrd', initrd, @token)
+      @server.call('modify_distro', distro_id, 'breed', breed, @token)
+      @server.call('save_distro', distro_id, @token)
+    rescue
+      raise 'creating distribution failed.' + $ERROR_INFO.to_s
```

Comment:
I think it's not a good practice to raise an exception in a rescue section.
What about just `puts` this error?

Reply:
We can use `puts` here of course. I just stuck to the same style the other methods in this class already used. Nearly all of them use `raise` inside a `rescue`.

Reply:
Ok to keep the same rescue management on this PR.